client/my-sites/customer-home/cards/features/go-mobile/style.scss

```diff
@@ -6,9 +14,16 @@
 	width: 50%;
 }
 
+.go-mobile__email-link {
+	margin: 24px 0 0;
+	padding-top: 20px;
+	border-top: 1px solid var( --color-neutral-5 );
+	color: var( --color-text-subtle );
+}
+
 .go-mobile__email-link-button {
-	margin-top: 1rem;
-	width: 100%;
+	cursor: pointer;
```

Comment:
Seems the button is displayed inline, causing a misplacement:

<img width="332" alt="Screen Shot 2020-05-08 at 10 42 41" src="https://user-images.githubusercontent.com/1233880/81388883-3fded700-9119-11ea-86f6-c0763b07a854.png">

Can be solved by setting `display: block;`:

<img width="344" alt="Screen Shot 2020-05-08 at 10 47 35" src="https://user-images.githubusercontent.com/1233880/81388984-64d34a00-9119-11ea-8e4d-9e797af41e3e.png">